From 681b4577eabf556d589a16ca774c507913d7054a Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Wed, 8 Apr 2015 15:23:39 +0200
Subject: [PATCH] virt-viewer-app: Do not show error dialog twice for unknown
 graphic

Related: rhbz#1085216
(cherry picked from commit 07def4d4aa9113911177bf3f51c1f1e3dde7eaab)
---
 src/virt-viewer-app.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/virt-viewer-app.c b/src/virt-viewer-app.c
index a688461..9468ca0 100644
--- a/src/virt-viewer-app.c
+++ b/src/virt-viewer-app.c
@@ -1081,8 +1081,6 @@ virt_viewer_app_create_session(VirtViewerApp *self, const gchar *type, GError **
 
         virt_viewer_app_trace(self, "Guest %s has unsupported %s display type",
                               priv->guest_name, type);
-        virt_viewer_app_simple_message_dialog(self, _("Unknown graphic type for the guest %s"),
-                                              priv->guest_name);
         return FALSE;
     }
 
